conf/nginx.conf

```diff
@@ -17,7 +17,7 @@ location __PATH__/ {
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection $http_connection;
 
-  client_max_body_size 100M;
+  client_max_body_size 10G;
```

Comment:
Do you think 10G is a good value?
I doubt there will we so many audiobooks with value of more than 1G (the issue comes here for more than 100M for the first time)...

But if you think it's a good definitive value, then I'm okay with it :)

Reply:
Yeah you are probably right, I am just matching it with Nextcloud's setting